package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@openhistoricalmap/map-styles",
+  "version": "0.9.2",
+  "description": "A collection of map styles for OpenHistoricalMap",
+  "main": "''",
+  "type": "module",
+  "scripts": {
+    "prepare": "scripts/prepare.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/OpenHistoricalMap/map-styles.git"
+  },
+  "author": "tsinn <tim@greeninfo.org>",
+  "contributors": [
+    "Vanessa Knoppke-Wetzel <run.for.funner@gmail.com>",
+    "Eric Theise <erictheise@gmail.com>"
+  ],
+  "license": "",
+  "bugs": {
+    "url": "https://github.com/OpenHistoricalMap/map-styles/issues"
```

Comment:
We should direct people to the central issues repo, https://github.com/OpenHistoricalMap/issues/issues. I thought we had disabled issues on all these other repos. When we had issues on each repo, we forever had vector tile issues or timeslider issues posted in map-styles, map-style issues posted in ohm-website, etc.

Reply:
Good catch. I think that's the default value when you run the `npm init` script and I was too busy wondering if the script would handle a null string for `license`. I'll fill in the license and correct the url.

Reply:
Probably better to use `https://github.com/OpenHistoricalMap/issues` as it puts people at the overview for our slighlty non-standard approach to issues/bugs. @danrademacher, do you want to be on the contributor list for the module?

Reply:
No need for me to be on the list